editor/import/resource_importer_texture_atlas.cpp

```diff
@@ -210,14 +211,18 @@ Error ResourceImporterTextureAtlas::import_group_file(const String &p_group_file
 		pack_data.is_cropped = options["crop_to_region"];
 
 		int mode = options["import_mode"];
+		bool trim_alpha_border_from_region = options["trim_alpha_border_from_region"];
 
 		if (mode == IMPORT_MODE_REGION) {
 			pack_data.is_mesh = false;
 
 			EditorAtlasPacker::Chart chart;
 
-			//clip a region from the image
-			Rect2 used_rect = image->get_used_rect();
+			Rect2 used_rect = Rect2(0, 0, image->get_width(), image->get_height());
+			if (trim_alpha_border_from_region) {
+				//clip a region from the image
```

Comment:
While at it, the comment style could be improved too:
```suggestion
				// Clip a region from the image.
```